src/components/My_profile/Profile.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './profile.css';
+import { PopulateMissionProfile } from '../../redux/mission/mission';
+import { PopulateRocketsProfile } from '../../redux/rockets/rocket';
+
+const Profile = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions);
+  const rockets = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    dispatch(PopulateMissionProfile());
+    dispatch(PopulateRocketsProfile());
+  }, []);
+
+  return (
+    <div className="myProfile">
+      <section className="my-comp">
+        <h2>My Mission</h2>
+        <ul className="reserved-container">
+          { missions.map((mission) => (mission.reserved === true ? (
+            <li className="reserved-item" key={mission.mission_id}>
+              {' '}
+              {mission.mission_name}
+              {' '}
+            </li>
```

Comment:
- [OPTIONAL] 🤔  What do you think if the My Profile section is enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved)? ✍🏻 

- [OPTIONAL] 🤔 What if the My Profile section is enhanced by adding the "Cancel reservation" and "Leave Mission" buttons to the lists here? Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections. 👍🏻 

- [OPTIONAL] The My Profile section can also be enhanced by adding the "Read more" button for each mission and rocket. Upon click, it should open a corresponding Wikipedia page in a new tab. NOTE - you need to get that extra Wikipedia URL from the API's payload. 👍🏻